examples/imaging/image_addition.py

```diff
@@ -36,10 +36,10 @@
 stars = Stars(
     grid.log10age,
     grid.metallicity,
-    instant_sf=10.,
-    instant_metallicity=0.01,
+    metal_dist=10.0,
```

Comment:
duplication?

Reply:
Surprised that wasn't picked up in the linting!

Reply:
Fixed now